### ISSUE TITLE
Help measuring larvae

### ISSUE THREAD

**Issue by laurahspencer** [RobertsLab/resources#725]
@kaitlynrm I have another batch of larvae that I need to get length/width measurements for. I'm thinking that we could ask Jackie to use her scope & computer for the task, which may allow for automatic measurements using the fancy software.  Larvae are currently in ethanol.  Hoping to get this done within then next few weeks, if possible.  Are you available to help?

**Comment by kaitlynrm** [comment]
Yes I can help although I am a bit sick currently (day 6 of not speaking...). Next week will work for me though. I think the automatic measurements will detect and measure the larvae, but I dont believe the software will consistently take the longest measurement because it is designed for round cells where as the larvae is bilateral.

**Comment by laurahspencer** [comment]
Great! Whenever you are available, let me know and we can schedule a time to meet for setup. 

I played around with FIJI (new ImageJ), and found that it's possible to adjust the color of larval images, fit ellipses around some of the larvae, and it automatically measures length and width. I never successfully got it to measure all larvae in the image, but I think it can be done.  The key is to get an accurate scale bar on all the images. I found that using a hemocytometer's grid as the scale sometimes resulted in larvae being at different depths, thus it wasn't reliable. Either Jackie's fancy computer/scope, OR our microscope with the eyepiece reticle would work.

**Comment by kaitlynrm** [comment]
How does Monday work for you?

**Comment by laurahspencer** [comment]
Any day this week works!

**Comment by kaitlynrm** [comment]
I saw you spent some time at Jackie's scope today. Did you figure out how to get it to automatically count?
